### ISSUE TITLE
Carousel - Cannot Cut and Paste

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#702]
## Bug Description

Unable to change order of pages by cut-and-paste

Taken from Claudio in #583 

## Screenshots

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by pbakaus** [comment]
agree. I know we had cut/copy/paste of pages in the old editor though. It's nice to have for me, but would require a "selected" state for the page itself.

**Comment by samitron7** [comment]
I didn't realize mod+L/R is working. Not too discoverable so we'll need to add that to the keyboard shortcut menu (the list that is display when you click on the keyboard icon).

Cut and paste is just more natural and doesn't require education but I do agree it's a nice to have feature because you can drag to reorder. We can really de-prioritize this ticket
